bundles/org.openhab.binding.irobot/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<binding:binding id="irobot" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:binding="https://openhab.org/schemas/binding/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
+
+	<name>iRobot Binding</name>
+	<description>This is the binding for iRobot vacuum robots.</description>
+	<author>Pavel Fedin</author>
```

Comment:
In the meanwhile this tag has been removed from all bindings. See #9113
```suggestion
```

Reply:
Fixed